README.md

```diff
@@ -112,6 +111,7 @@ Options:
   -p, --postmanUid           Postman collection UID to upload with the generated Postman collection           [string]
       --postmanWorkspaceName Postman Workspace name to target the upload of the generated Postman collection  [string]
   -t, --includeTests         Inject Portman test suite (default: true)                                        [boolean]
+      --ignoreCircularRefs   Ignore circular references in OpenAPI spec (default: false)                      [boolean]
```

Comment:
I would put it right above "--init" since they were a bit grouped per logic coherence and now it sits in between includeTests and the related bundleContractTests